src/FileSystem/FilesFinder.php

```diff
@@ -68,7 +68,11 @@ private function findInDirectories(array $directories, array $suffixes): array
 
         $filePaths = [];
         foreach ($finder as $fileInfo) {
-            $filePaths[] = $fileInfo->getRealPath();
+            /** @var string|false $path */
+            $path = $fileInfo->getRealPath();
+            if ($path !== false) {
+                $filePaths[] = $path;
+            }
```

Comment:
Could you add a comment about symlinks here, so we know "why" this is needed?
We had some cleanups about real paths in the past, so we stick with the reason :)

Reply:
Sounds like the return type of the method beeing called is wrong?

Reply:
That's seems PHPStan internal bug false positive for SplFileInfo